### PR TITLE
Upgrade to rails 6 step 3

### PR DESCRIPTION
**Pull request by matt-hh**

### CODE REVIEW

File: Dockerfile.github-ci [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM ruby:2.6.8
```
Comment (JanCBrammer) [inline Dockerfile.github-ci:1]: Should this be `FROM ruby:2.7.6`? I.e., using another base image.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -1,17 +1,16 @@
 # This docker-compose file sets up the dev environment.
-# It builds an app container based on Dockerfile.chemotion.dev
+# It builds an app container based on Dockerfile.chemotion.de
```
Comment (JanCBrammer) [inline docker-compose.dev.yml:2]: Is the `de` intentional, or is it supposed to be `dev`?

**Review by JanCBrammer — COMMENTED** [review]
Looks good to me as far as the diffs are concerned. Next, I'll check out the branch and click through the app / do some manual testing.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -49,7 +49,7 @@ jobs:
       run:
         shell: bash
 
-    container: jancbrammer/chemotion-ci-runner:debian    # TODO: serve image from complat account
+    container: matthh/chemotion-ci-runner:ruby-2.7.6    # TODO: serve image from complat account
```
Comment (PiTrem) [inline .github/workflows/ci.yml:52]: complat/chemotion_eln_runner:ruby-2.7.6 is now available

Comment (matt-hh) [reply .github/workflows/ci.yml]: 👍 done

**Review by PiTrem — APPROVED** [review]

**Pull request merged**